system/Debug/Exceptions.php

```diff
@@ -293,16 +293,61 @@ protected function render(Throwable $exception, int $statusCode)
 	 */
 	protected function collectVars(Throwable $exception, int $statusCode): array
 	{
+		$trace = $exception->getTrace();
+		if(!empty($this->config->sensitiveDataInTrace)) {
+			$this->maskSensitiveData($trace, $this->config->sensitiveDataInTrace);
+		}
+		
 		return [
 			'title'   => get_class($exception),
 			'type'    => get_class($exception),
 			'code'    => $statusCode,
 			'message' => $exception->getMessage() ?? '(null)',
 			'file'    => $exception->getFile(),
 			'line'    => $exception->getLine(),
-			'trace'   => $exception->getTrace(),
+			'trace'   => $trace,
 		];
 	}
+	
+	/**
+	 * Mask sensitive data in the trace.
+	 *
+	 * @param array  $trace
+	 * @param array  $keysToMask
+	 * @param string $path
+	 */
+	protected function maskSensitiveData(&$trace, $keysToMask, $path = '') 
```

Comment:
```suggestion
	protected function maskSensitiveData(array &$trace, array $keysToMask, string $path = '') 
```

Reply:
Setting $keysToMask, and $path to string type is fine, but $trace can either be an array, or object

Reply:
> Can you show a sample output using this ?

Putting the whole trace would be huge, but here's a part of it for masking 'app/settings'

 [app] => App\Models\SystemModel Object
        (
            [settings] => ******************
            [currencies] => Array
                (
                )

            [currencyFormatter] => 
            [table:protected] => 
            [primaryKey:protected] => id
            [useAutoIncrement:protected] => 1
            [builder:protected] => 
            [tempData:protected] => Array
                (
                )

            [escape:protected] => Array
                (
                )

Reply:
> Setting $keysToMask, and $path to string type is fine, but $trace can either be an array, or object

OK. So put `array|object` in the docblock